src/hist/named.py

```diff
@@ -21,18 +41,21 @@ def fill(self, *args, **kwargs):
         indices = []
         values = []
         for name, val in kwargs.items():
+            find = False
             for index, axis in enumerate(self.axes):
                 if name == axis.name:
                     indices.append(index)
                     values.append(val)
+                    find = True
                     break
 
+            if not find:
```

Comment:
You can actually just make this an else class *on the for loop*, and drop the find variable. The else clause of a for loop runs when the loop was never broken.

Reply:
https://book.pythontips.com/en/latest/for_-_else.html#else-clause